nf_core/utils.py

```diff
@@ -489,37 +489,44 @@ def get_biocontainer_tag(package, version):
         A ValueError, if the package name can not be found (404)
     """
 
-    def get_tag_date(tag_date):
-        # Reformat a date given by quay.io to  datetime
-        return datetime.datetime.strptime(tag_date.replace("-0000", "").strip(), "%a, %d %b %Y %H:%M:%S")
+    biocontainers_api_url = f"https://api.biocontainers.pro/ga4gh/trs/v2/tools/{package}/versions/{package}-{version}"
 
-    quay_api_url = f"https://quay.io/api/v1/repository/biocontainers/{package}/tag/"
+    def get_tag_date(tag_date):
+        """
+        Format a date given by the biocontainers API
+        Given format: '2021-03-25T08:53:00Z'
+        """
+        tag_date = tag_date.split("T")[0].split("-")
+        return datetime.date(year=int(tag_date[0]), month=int(tag_date[1]), day=int(tag_date[2]))
```

Comment:
I think this is an `ISO 8601` date string right? I wonder if something like this would be nicer: https://stackoverflow.com/a/3908349/713980

```python
import dateutil.parser
yourdate = dateutil.parser.parse(datestring)
```

Reply:
Thanks, I'll try that!